obf-generation/travel-gpx/README.md

```diff
@@ -0,0 +1,144 @@
+GPX=osmand:vertical_exaggeration_scale OBF=gpx_vertical_exaggeration_scale POI,FLEXIBLE
+```
+
+### 2.2. Waypoints
+
+```
+GPX=osmand:color OBF=color POI,MAP,STYLE,SPECIAL
+GPX=osmand:icon OBF=gpx_icon POI,STYLE
+GPX=osmand:background OBF=gpx_bg POI,STYLE
+GPX=<type> OBF=points_groups_category POI
+```
+
+## 3. Save/Restore (shields and OSM-related tags)
+
+```
+GPX=osmand:osm_tag_* OBF=[POI|MAP] POI,MAP,STYLE,FLEXIBLE (some OSM tags will be lost: osmc:symbol, route, type, etc)
+
+GPX=osmand:use_osmc_colors OBF=use_osmc_colors POI,MAP,STYLE
+
+GPX=osmand:osm_id OBF=osm_id POI,MAP (will be used to keep/display OSM id if applicable)
+GPX=osmand:osm_ref_present OBF=gpx_osm_ref_present POI ("yes" if "ref" comes from osm tags)
+GPX=osmand:osm_name_present OBF=gpx_osm_name_present POI ("yes" if "name" comes from osm tags)
+GPX=osmand:osm_description_present OBF=gpx_osm_description_present POI ("yes" if "description" comes from osm tags)
+
+GPX=osmand:shield_bg OBF=shield_bg POI,MAP,STYLE
+GPX=osmand:shield_fg OBF=shield_fg POI,MAP,STYLE
+GPX=osmand:shield_fg_2 OBF=shield_fg_2 POI,MAP,STYLE
+GPX=osmand:shield_text OBF=shield_text POI,MAP,STYLE
+GPX=osmand:shield_textcolor OBF=shield_textcolor POI,MAP,STYLE
+GPX=osmand:shield_waycolor OBF=shield_waycolor POI,SPECIAL (shadow copy of osmc_waycolor)
+```
+
+## 4. Save/Restore (analytics, elevation and special tags)
+
+```
+OBF=route_id POI,MAP,SPECIAL (used to identify unique GPX and to connect Map<->POI data)
+
+OBF=route_radius POI (specify default radius to search in POI section)
+
+OBF=distance POI (distance of all GPX segments)
+OBF=max_speed,avg_speed,min_speed POI (speed analytics)
+OBF=time_span,time_span_no_gaps,time_moving,time_moving_no_gaps POI (time analytics)
+OBF=start_ele,ele_graph MAP (GPX-specified elevation data wrapped in compact binary array)
+OBF=min_ele,avg_ele,max_ele,diff_ele_up,diff_ele_down POI,MAP (analytics of GPX-specified elevation data)
+```
+
+## 5. Rendering (Map-section)
+
+### 5.1. Tracks
+
+```
+GPX=<trkseg> OBF=route VAL=segment
+
+GPX=osmand:color OBF=color POI,MAP,STYLE,SPECIAL
+GPX=osmand:width OBF=gpx_width POI,MAP,STYLE (fixed width values supported)
+
+GPX=osmand:shield_bg OBF=shield_bg POI,MAP,STYLE
+GPX=osmand:shield_fg OBF=shield_fg POI,MAP,STYLE
+GPX=osmand:shield_fg_2 OBF=shield_fg_2 POI,MAP,STYLE
+GPX=osmand:shield_text OBF=shield_text POI,MAP,STYLE
+GPX=osmand:shield_textcolor OBF=shield_textcolor POI,MAP,STYLE
+
+GPX=osmand:use_osmc_colors OBF=use_osmc_colors POI,MAP,STYLE (transparent colors)
+```
+
+### 5.2. Waypoints
+
+```
+GPX=<wpt> OBF=route VAL=point
+GPX=osmand:icon OBF=gpx_icon POI,STYLE
+GPX=osmand:color OBF=color POI,MAP,STYLE,SPECIAL
+GPX=osmand:background OBF=gpx_bg POI,STYLE (mix color+gpx_bg is badly supported due to lack of icons)
+```
+
+## 6. Categories/Search (POI-section)
+
+### 6.1. POI-categories (search, filters, show-on-map)
+
+```
+GPX=osmand:osm_tag_route OBF=route_type VALUE=cycling POI (Bicycle)
```

Comment:
https://github.com/osmandapp/OsmAnd-Issues/issues/2758 - activity is stored different way in GPX
- Move activity tag to metadata extensions - https://www.topografix.com/gpx/1/1/#type_metadataType (osmand:activity)